dockertest/subtest.py

```diff
@@ -410,7 +410,11 @@ def try_all_stages(self, name, subsubtest):
                               self.exception_info["error_source"],
                               detail)
             exc_info = self.exception_info["exc_info"]
-            # cleanup() will still be called before this propigates
+            # Treat N/A as passed, even though this may hide obscure failures
+            if isinstance(detail, DockerTestNAError):
+                self.logdebug("Treating DockerTestNAError as PASS")
+                self.final_subsubtests.add(name)
```

Comment:
Ahh ok, good, ya nice to have that message, it also saves needing a comment :smile: (for the ``final_subsubtests.add()``)